### ISSUE TITLE
custom blog post template?

### ISSUE THREAD

**Issue by jtoy** [greyblake/blogo#37]
is this possible? there is no info in the docs or wiki

**Comment by jtoy** [comment]
i see how to do it, this should definitely be in the docs somewhere so that other people can do this.

**Comment by greyblake** [comment]
Yes, it's possible, you can redefine existing templates in you app
